smug/run.py

```diff
@@ -28,6 +28,7 @@ def run():
         processes.append(subprocess.Popen(['python', 'preprocessors/preprocessing.py']))
     for i in range(processors):
         processes.append(subprocess.Popen(['python', 'processors/word_vectoring_processor.py']))
+        processes.append(subprocess.Popen(['python', 'processors/location_prediction_module.py']))
```

Comment:
should have it's own for loop and variable. the location prediction may need another amount of processes than word vector